alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -320,19 +321,45 @@ private function getIfarmEmbedCode($bundleContent)
                     </html >';
 		return $htmlDoc;
 	}
+
+	private function toAssociativeArray($input)
+	{
+		$configs = explode(",", $input);
+		$arr = array();
+		foreach($configs as $conf)
+		{
+			$obj = explode("=", $conf);
+			$key = $obj[0];
+			$value = $obj[1];
+			$arr[$key] = $value;
+		}
+		return $arr;
+	}
+
+	private function mergeVersionsParamIntoConfig()
+	{
+		//Get version from QS
+		$versions = $this->getRequestParameter(self::VERSIONS_PARAM_NAME);
+		$pattern = '/[^?&,]+=[^?&,]+(?>,[^,?&]+=[^,?&]+)*/'; // key value object
+		preg_match($pattern, $versions, $matches);
```

Comment:
preg_match returns a value that indicates if it succeeded or not.
Running strlen($matches[0]) in cased it failed will result in notice.